src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -255,6 +262,7 @@ public function request(string $method, string $url, array $options = []): Respo
                 throw new TransportException(sprintf('Curl option "%s" is not supported.', key($constants) ?? $opt));
             }
         }
+        $this->logger->debug('Creating response', ['response_class' => CurlResponse::class]);
```

Comment:
I don't think this log message is useful. It will always be the same for all responses, following the more useful `Making a request` which contains the URL and method.